src/kubernetes-ingress-controller/references/cli-arguments.md

```diff
@@ -52,7 +52,10 @@ Following table describes all the flags that are available:
 | `--enable-controller-tcpingress`                | `boolean` | Enable the TCPIngress controller.                                                                     | `true`                         |
 | `--enable-controller-udpingress`                | `boolean` | Enable the UDPIngress controller.                                                                     | `true`                         |
 | `--enable-reverse-sync`                  | `boolean`          | Send configuration to Kong even if the configuration checksum has not changed since previous update.                                                | `false`                           |
-| `--health-probe-bind-address`            | `string`           | The address the probe endpoint binds. to.                                                                                                            | `":10254"`                        |
+{% if_version gte 2.6.x %}
+| `--gateway-api-controller-name`          | `string`           | Controller name of gateway API. `Gateway` resources are reconciled only when their `GatewayClass` has same value in `spec.controllerName`.           | `konghq.com/kic-gateway-controller` |
```

Comment:
```suggestion
| `--gateway-api-controller-name`          | `string`           | Controller name of the Kong Gateway API. `Gateway` resources are reconciled only when their `GatewayClass` has the same value in `spec.controllerName`.           | `konghq.com/kic-gateway-controller` |
```

Reply:
`gateway API` here means the kubernetes gateway API: https://github.com/kubernetes-sigs/gateway-api. It is not the Kong Gateway API. How about changing to `kubernetes gateway API`  here?